schedule/yast/multipath.yaml

```diff
@@ -4,34 +4,19 @@ description:    >
   Tests successful detection of multipath and installation and
   validate multipath configuration and tools after booting.
 vars:
-  DESKTOP: gnome
   MULTIPATH: 1
   YUI_REST_API: 1
 schedule:
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - installation/product_selection/install_SLES
-  - installation/licensing/accept_license
-  - installation/registration/register_via_scc
-  - installation/module_registration/register_module_desktop
-  - installation/multipath
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_SLES_with_GNOME
-  - installation/partitioning/accept_proposed_layout
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/use_same_password_for_root
-  - installation/authentication/default_user_simple_pwd
-  - installation/installation_settings/validate_default_target
-  - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/launch_installation
-  - installation/confirm_installation
-  - installation/performing_installation/perform_installation
-  - installation/logs_from_installation_system
-  - installation/performing_installation/confirm_reboot
-  - installation/grub_test
-  - installation/first_boot
-  - console/system_prepare
-  - console/force_scheduled_tasks
-  - console/validate_multipath
-test_data:
-  <<: !include test_data/yast/multipath.yaml
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  multipath:
```

Comment:
could you please file a ticket to remove `multipath` key and replace reuse `system_probing` for all archs?

Reply:
> could you please file a ticket to remove `multipath` key and replace reuse `system_probing` for all archs?

OK

Reply:
I file a ticket for this: https://progress.opensuse.org/issues/132266